src/openMVG/sfm/sfm_data_io_gt.hpp

```diff
@@ -119,35 +126,47 @@ static bool read_Strecha_Camera(const std::string & camName, Pinhole_Intrinsic &
 
 /**
 @brief Reads a set of Pinhole Cameras and its poses from a ground truth dataset.
-@param[in] functorPointer, to the function which can handle the trajectory format. Example: &read_openMVG_Camera
 @param[in] sGTPath, the directory where the camera files are located.
```

Comment:
Good solution, but I have to add this input parameters in `sfm::Load()` also : https://github.com/poparteu/openMVG/blob/7f7fe6a41be23402dc4c2422ebc5ca22d7aaf2a9/src/openMVG/sfm/sfm_data_io.cpp#L93